.gitignore

```
@@ -0,0 +1 @@
+.vscode/**
```

Buna gerek var mıydı?
```suggestion
.vscode
```